src/views/catalog/utils/quantity.ts

```diff
@@ -15,8 +15,8 @@ export const bytesToIECBytes = (
 };
 
 export const bytesToDiskSize = (size: string) => {
-  const bytesizeresult = bytesToIECBytes(parseFloat(size), 0, binaryUnits);
-  return [bytesizeresult.value, bytesizeresult.unit].join('');
+  const bytesizeresult = bytesFromQuantity(size, 0);
+  return [bytesizeresult[0], bytesizeresult[1]].join('');
```

Comment:
1. is an array in size of 2 guaranteed? if not please add ?.[x]

2. can u simply return bytessizeresult? or does it contain more items, and u want only the first 2?

Reply:
1. it is guaranteed (and we always do want and need both items, not only one)
2. no, we need to join the items, we need to return a string. If I edited `bytesFromQuantity` itself for our needs here, it would break some other places in the UI.

Reply:
or i see, so you are using join instead of toString here, ok, it not really matter.

Reply:
`bytesizeresult.toString()` returns `30,GiB` (for example) which is not what we want. We want `30GiB` instead. And from what I know, `toString` does not take any params. So we really don't have any other option than to use `join` here.